transport/http/transport.go

```diff
@@ -281,10 +280,11 @@ func (o *transportOptions) newTransport() *Transport {
 		meter:                    o.meter,
 		serviceName:              o.serviceName,
 		ouboundTLSConfigProvider: o.outboundTLSConfigProvider,
+		h1Transport:              defaultH1Transport(o),
 	}
 }
 
-func buildHTTPClient(options *transportOptions) *http.Client {
+func defaultH1Transport(options *transportOptions) *http.Transport {
```

Comment:
Why is this called "defaultH1Transport"? It seems like it's always creating new transport based on a passed options, i.e. it's not default, it's a properly configured transport, right?

Reply:
yep, it should be renamed to buildH1Transport instead. Will update it.